pkgs/development/libraries/jcal/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, which, libtool, getopt, readline}:
+
+stdenv.mkDerivation rec {
+  name = "jcal";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "fzerorubigd";
+    repo = "jcal";
+    rev = "v${version}";
+    sha256 = "1b2nm499wcbpp2xim30b6sazkcg901cfr75qigclschj5rns6s9w";
```

Comment:
This hash is incorrect now that the release tarball is being used. You probably didn't see this because you already had the old one downloaded under this hash (a bit of a common gotcha with fixed-output derivations). It should be `0m3g3rf0ycv2dsfn9y2472fa3r0yla8pfqk6gq00nrscsc3pp4zf` now, which you can verify by putting in a nonexistent hash :)